packages/multiple-did-identities/src/utils/generateKeyPairs.ts

```diff
@@ -39,3 +39,95 @@ export const generateKeyPairs = async (numKeys: number) => {
 
   return keys;
 };
+
+// Function to generate multiple ED25519 key pairs and include raw and JWK format
+export const generateKeyPairsED25519 = async (numKeys: number) => {
+  const pubKeyType = 'JsonWebKey2020';
+  const keys: Array<{
+    rawPublicKey: Uint8Array;
+    rawPrivateKey: Uint8Array;
+    publicKeyJwk: JWKKeys;
+    privateKeyJwk: PrivateKeyJWK;
+  }> = [];
+
+  for (let i = 0; i < numKeys; i++) {
+    // Generate private and public keys
+    const privateKey = ed25519.utils.randomPrivateKey();
+    const publicKey = await ed25519.getPublicKey(privateKey);
+
+    // Create JWK for public and private keys
+    const publicKeyJwk: JWKKeys = {
```

Comment:
Instead of hardcoding the structure of publicKeyJwk and privateKeyJwk, you could dynamically validate them against a schema right ?.